src/server/req_manager.c

```diff
@@ -4579,8 +4582,14 @@ req_manager(struct batch_request *preq)
 					break;
 
 				case MGR_OBJ_SCHED:
-					if (preq->rq_ind.rq_manager.rq_cmd == MGR_CMD_CREATE)
+					if (preq->rq_ind.rq_manager.rq_cmd == MGR_CMD_CREATE) {
+						if ( obj_name_len == 0 ) {
+							preq->rq_ind.rq_manager.rq_objname[PBS_MAXSVRJOBID] = '\0';
```

Comment:
This needs to go after the strncpy() (this goes for the other two instances in this file)